docusaurus.config.js

```diff
@@ -42,7 +45,8 @@ const config = {
           path: "docs",
           // @ts-ignore
           // eslint-disable-next-line global-require
-          remarkPlugins: [require("remark-docusaurus-tabs")],
+          remarkPlugins: [require("remark-docusaurus-tabs"), math],
+          rehypePlugins: [katex],
```

Comment:
@yelhousni can you check this works fine on the vercel preview? (see PR in github)